### PR TITLE
[RNBW-2749] Skeleton flashes

### PR DESCRIPTION
**Pull request by perunt**
Fixes RNBW-2749

## What changed (plus any additional context for devs)
@react-native-community/masked-view is outdated, updated into favor react-native-masked-view/masked-view
Removed in/change transition animation case it looks like a hack for the old version of reanimated

## PoW (screenshots / screen recordings)
https://recordit.co/oxKJ6c2otK

https://recordit.co/mcEydiqO3J
## Dev checklist for QA: what to test
Skeleton in Discover search screen

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: src/components/exchange/CurrencySelectionList.js
Comment (terrysahaidak) [inline src/components/exchange/CurrencySelectionList.js:31]: Could you double-check this was useless?

**Comment by osdnk** [comment]
Can you check if works for loading wallet or switching?

**Comment by tchayen** [comment]
I think maybe it'd be better to have QA check on this PR before merging rather then waiting for release?

**Comment by osdnk** [comment]
I don't have strong opinions. I think it's pretty safe after testing m

**Comment by perunt** [comment]
Transitions API deprecated and now doesn't work correctly on Android.
I tried to rewrite the fade animation using a newer version of Reanimated but was faced with spoilage in performance for Android.
I've been looking for other ways to fix the animation but haven't found the right one yet
Since in this place this animation worked only for the iOS platform, as a temporary workaround I decided to do the same as in the neighbor element (WalletList), making it optional only for ios. So in the end, these splashes were fixed but we need to replace Transitions API

**Review by terrysahaidak — APPROVED** [review]

**Review by osdnk — APPROVED** [review]
ok

**Review by tchayen — APPROVED** [review]

**Pull request merged**
